src/components/AgendaItems/AgendaItemsCreateModal.spec.tsx

```diff
@@ -19,6 +19,10 @@ import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import AgendaItemsCreateModal from './AgendaItemsCreateModal';
 import { toast } from 'react-toastify';
 import convertToBase64 from 'utils/convertToBase64';
+import type { MockedFunction } from 'vitest';
+
+import '@testing-library/jest-dom';
```

Comment:
Why was this added if we are migrating from Jest?

Reply:
I was going though the guide I though it is necessary but its not.
I have added screenshot as well it works after removal
![lastChange](https://github.com/user-attachments/assets/807e5377-c8c2-4e95-8c9f-9066e94a0be6)